### ISSUE TITLE
Support check `.unshift()` in `no-array-push-push`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1019]
`no-array-push-push` was added in #1015, only check `.push()`, opening a new issue to see if someone need this.

## Fail

```js
foo.unshift(1);
foo.unshift(2);
```

### Pass

```js
const length = foo.unshift(1);
foo.unshift(2);
```

Note: `.unshift()` is a little different, need change arguments order, and consider side effects in each call.(`.push` only consider side effect in second call).

Original issue #937

**Comment by sindresorhus** [comment]
Sounds good 👍
